android/app/src/main/java/com/wix/reactnativenotifications/RNNotificationsModule.java

```diff
@@ -46,8 +46,22 @@ public String getName() {
     @Override
     public void initialize() {
         Log.d(LOGTAG, "Native module init");
-        startFcmIntentService(FcmInstanceIdRefreshHandlerService.EXTRA_IS_APP_INIT);
-
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
```

Comment:
Can we add a TODO here to remove this code once the Android team releases their fix for this issue?

Reply:
We shouldn't remove this as there will be Android users on older versions.